gensim/models/base_any2vec.py

```diff
@@ -476,15 +551,21 @@ def build_vocab(self, sentences, update=False, progress_per=10000, keep_raw_voca
             Indicates how many words to process before showing/updating the progress.
 
         """
+        start_time = time.time()
+
         total_words, corpus_count = self.vocabulary.scan_vocab(
-            sentences, progress_per=progress_per, trim_rule=trim_rule)
+            input_streams, progress_per=progress_per, workers=workers, trim_rule=trim_rule)
```

Comment:
How we'll distinguish `input_streams` and `input_stream` (we should maintain backward compatibility here)?